client/my-sites/site-settings/form-general.jsx

```diff
@@ -170,6 +171,23 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	toolbarOption() {
+		const { isRequestingSettings, isSavingSettings, siteIsJetpack, siteIsAtomic } = this.props;
+
+		const isNonAtomicJetpackSite = siteIsJetpack && ! siteIsAtomic;
+
+		return (
+			<>
+				{ isNonAtomicJetpackSite && (
+					<Masterbar
```

Comment:
Rendering `Masterbar` in the `SiteSettingsFormGeneral` will break its tests, because we also need to provide the Jetpack reducer when rendering the component with `@testing-library/react`.

To do that, you should:
1. `import jetpackReducer from 'calypso/state/jetpack/reducer';` in `./test/form-general.jsx`
2. Provide `jetpack: {},` to the `initialState` constant.
3. Provide `jetpack: jetpackReducer,` to the list of reducers in `renderWithRedux()`.

In order to run these tests locally, you can run this command in your terminal:

```
yarn run test-client client/my-sites/site-settings/test/form-general.jsx
```

That should do the job, but let me know if you're having any trouble.